packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -717,7 +732,16 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
       );
     }
 
-    return _configuration.buildViewportChrome(context, result, widget.axisDirection);
+    return _configuration.useDecoration
```

Comment:
It's unfortunate that `useDecoration` this leaks out of the ScrollConfiguration. Ideally, this would just unconditionally call the new method and internally it hushes out what needs to happen.

Reply:
Could we just unconditionally call `buildViewportDecoration` and internally ScrollConfiguration.buildViewportDecoration checks whether the flag is set and calls the other method?

Reply:
Oh yes, that sounds much cleaner.